packages/lib-utils/src/components/table-view/TableView.tsx

```diff
@@ -25,14 +25,14 @@ export type FilterItem = {
   id: string;
 };
 
-export type TableViewProps<D> = VirtualizedTableProps<D> & {
+export type ListViewProps<D> = VirtualizedTableProps<D> & {
```

Comment:
@fhlavac I'm wondering if we should update the name of the files/directories as well to avoid confusion?

Reply:
Sure, it's updated. Thank you :slightly_smiling_face: